### PR TITLE
*: add codegen for pkg/build/build.sh

### PR DESCRIPTION
**Pull request by fanminshi**
cc/ @hasbro17

### CODE REVIEW

**Comment by fanminshi** [comment]
Test:
```sh
fanmins-MacBook-Pro:operator-framework fanminshi$ operator-sdk new test-operator
Create /Users/fanminshi/work/src/github.com/operator-framework/test-operator/cmd/manager/main.go
Create /Users/fanminshi/work/src/github.com/operator-framework/test-operator/pkg/apis/apis.go
Create /Users/fanminshi/work/src/github.com/operator-framework/test-operator/pkg/controller/controller.go
Create /Users/fanminshi/work/src/github.com/operator-framework/test-operator/pkg/build/Dockerfile
Create /Users/fanminshi/work/src/github.com/operator-framework/test-operator/pkg/build/build.sh
Run dep ensure ...
could not find project Gopkg.toml, use dep init to initiate a manifest
2018/09/14 15:50:34 failed to exec dep []string{"ensure", "-v"}: exit status 1
fanmins-MacBook-Pro:operator-framework fanminshi$ cat test-operator/pkg/build/build.sh
#!/usr/bin/env bash

set -o errexit
set -o nounset
set -o pipefail

if ! which go > /dev/null; then
	echo "golang needs to be installed"
	exit 1
fi

BIN_DIR="$(pwd)/build/_output/bin"
mkdir -p ${BIN_DIR}
PROJECT_NAME=test-operator
REPO_PATH=github.com/operator-framework/test-operator
BUILD_PATH="${REPO_PATH}/cmd/manager"
echo "building "${PROJECT_NAME}"..."
GOOS=linux GOARCH=amd64 CGO_ENABLED=0 go build -o ${BIN_DIR}/${PROJECT_NAME} $BUILD_PATH
```

File: commands/operator-sdk/cmd/new.go [hunk 1]
```diff
@@ -179,6 +181,23 @@ func doScaffold() {
 		log.Fatalf("failed to create %v: %v", dockerfilePath, err)
 	}
 
+	// generate pkg/build/build.sh
+	buildScriptPath := filepath.Join(buildDir, "build.sh")
+	buildScriptgen := scaffold.NewbuildCodegen(
```
Comment (hasbro17) [inline commands/operator-sdk/cmd/new.go]: Minor nit: Just noticed you might want to fix the camel case here and above in other places.
`buildScriptGen`

**Comment by hasbro17** [comment]
LGTM after nit.

File: commands/operator-sdk/cmd/new.go (hunk 1 above)
Comment (fanminshi) [reply commands/operator-sdk/cmd/new.go]: yeah, good catch!

**Pull request merged**